chaste_codegen/_command_line_script.py

```diff
@@ -6,26 +6,29 @@
 from collections import OrderedDict
 
 import chaste_codegen as cg
+from chaste_codegen._lookup_tables import _DEFAULT_LOOKUP_PARAMETERS

```

Comment:
I'm not sure it's a good idea to prefix with _ something that's designed to be imported?

Reply:
hhm good point